components/fullscreen-image-quote/Component.jsx

```diff
@@ -0,0 +1,112 @@
+ *
+ * @param {boolean} removeTopSpacing
+ * When this is true, the component will be shifted up to eliminate the spacing between itself and the component above.
+ * This is useful when several Fullscreen Image Quotes are stacked on top of each other.
+ *
+ * @param {object} imageData
+ * The data of the background image
+ *
+ * @param {string} imageVPosition
+ * The crop position (object-position) of the image: "top" | "center" | "bottom"
+ *
+ * @param {object} mobileImageData
+ * The data of the background image for mobile
+ *
+ * @param {string} internalLinkUrl
+ * The internal link URL
+ *
+ * @param {object} ctaDetails
+ * The different fields for the CTA
+ *
+ * @returns {JSX.Element}
+ * @constructor
+ */
+
+export default function FullscreenImageQuote({
+  quote,
+  quoteHAlign,
+  quoteVAlign,
+  removeTopSpacing,
+  imageData,
+  imageVPosition,
+  mobileImageData,
+  internalLinkUrl,
+  ctaDetails,
+}) {
+  const { ctaPreText, ctaText, ctaSubtext, externalUrl, isNewWindow } =
+    ctaDetails;
+  // Check externalUrl field to see if it is actually external
+  // Do not use external arrow icon and rel attributes if link has news.stanford.edu
+  const isRealExternalLink =
+    !!externalUrl && !externalUrl?.includes("news.stanford.edu");
+
+  return (
+    <Container
+      width="full"
+      paddingX={false}
+      className={styles.root(removeTopSpacing)}
+    >
+      <div className={styles.contentWrapper(quoteVAlign)}>
+        <blockquote className={styles.blockquote(quoteHAlign)}>
+          {quote && <p className={styles.quote}>{quote}</p>}
+          {ctaText && (
+            <div className={styles.cta}>
+              <span className={styles.ctaPreText}>{ctaPreText}</span>
+              <a
+                href={externalUrl || internalLinkUrl}
+                target={isNewWindow ? "_blank" : undefined}
+                rel={isRealExternalLink ? "noopener nofollow" : undefined}
+                className={styles.ctaLink}
+              >
+                {ctaText}
+                {/* Use this whitespace-nowrap trick so icon won't get pushed to the next line on its own */}
+                <span className={styles.ctaIconWrapper}>
+                  &#65279;
+                  <FAIcon
+                    width={18}
+                    icon={isRealExternalLink ? "arrow-up" : "chevron-right"}
+                    title={isRealExternalLink ? "link is external" : undefined}
+                    className={styles.ctaIcon(isRealExternalLink)}
+                  />
+                </span>
+              </a>
+              {ctaSubtext && (
+                <div className={styles.ctaSubtext}>{ctaSubtext}</div>
+              )}
+            </div>
+          )}
+        </blockquote>
+      </div>
+      {mobileImageData?.url && (
+        <img
+          src={mobileImageData?.url}
+          alt={mobileImageData?.attributes?.alt || ""}
+          className={styles.mobileImage}
+        />
+      )}
```

Comment:
The mobile image is optional. On the test page, only the 2nd and 3rd components have mobile images added.